service_supervisord_linux.go

```diff
@@ -0,0 +1,170 @@
+package service
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"text/template"
+	"time"
+)
+
+func isSupervisord() bool {
+	if _, err := os.Stat("/usr/bin/supervisord"); err == nil {
```

Comment:
I think adding ENV variable e.g. ENABLE_SUPERVISORD would be good idea. Then push `linux-supervisord` to the top of the list. This way you could use it for pmm-managed and also we can play with it with pmm-client as undocumented experiment.